README.md

```diff
@@ -15,6 +15,10 @@ The OpenAPI Specification does not require rewriting existing APIs. It does not
 
 This GitHub project is the starting point for OpenAPI. Here you will find the information you need about the OpenAPI Specification, simple examples of what it looks like, and some general information regarding the project.
 
+Here is an example API description file following the OpenAPI Specification: https://petstore.swagger.io/v2/swagger.json
+
+And here is an API documentation website rendered automatically from that file: https://petstore.swagger.io
```

Comment:
Because of the range of open-source and commercial solutions providing documentation renderers (as well as this only being one use-case for an OpenAPI document), I think it would be better to remove this entirely.